DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+      - 'True'
+      - 'False'
+    slice_thickness:
+      description: Thickness of the imaged slice in micrometres
+      type: string
+      format: um
+      ontology_link: https://bioportal.bioontology.org/ontologies/DCM?p=classes&conceptid=http%3A%2F%2Fdicom.nema.org%2Fresources%2Fontology%2FDCM%2F112225
+    internal_anatomical_structures:
+      description: Internal (landmark) structures visible in the overview image that
+        are informative about the broader anatomical context/location of the sample
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/OCHV?p=classes&conceptid=http%3A%2F%2Fsbmi.uth.tmc.edu%2Fontology%2Fochv%2325102
+    fiducial_marker:
+      description: Fiducial markers for the alignment of images taken across multiple
+        rounds of imaging
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C82602
+    expansion_factor:
+      description: Factor by which the imaged tissue was expanded in one dimension
+      type: integer
+      format: number
+      ontology_link: https://bioportal.bioontology.org/ontologies/AGRO?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FIAO_0000101
+    microscope_setup_description:
+      description: Description of the microscope setup
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/OBI?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0400169
+    microscopy_technique:
+      description: The type of microscopy
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/SNOMEDCT?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FSNOMEDCT%2F278289002
+    magnification:
+      description: Magnification of the objective used for imaging
+      type: string
+      format: text
+    numerical_aperture:
+      description: Numerical aperture of the objective
+      type: integer
+      format: number
+      ontology_link: https://bioportal.bioontology.org/ontologies/PLOSTHES?p=classes&conceptid=http%3A%2F%2Flocalhost%2Fplosthes.2017-1%2310222
+    immersion_medium_type:
+      description: Immersion medium used for imaging
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C182013
+    immersion_medium_refractive_index:
+      description: Refractive index of the immersion medium used for imaging
+      type: integer
+      format: number
+      ontology_link: https://bioportal.bioontology.org/ontologies/FBbi?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFBbi_00000311
+    pixel_size:
+      description: Pixel size in nanometers
+      type: integer
+      format: nm
```

Comment:
@ishitasati11 Let's separate format from unit of measurement wherever applicable